core/controllers/acl_decorators.py

```diff
@@ -2753,18 +2753,17 @@ def test_can_accept_suggestion(
     return generate_decorator_for_handler
 
 
-def can_update_suggestions(handler):
-    """Decorator to check whether user can update the suggestions that
-    they are allowed to review.
+def can_reviewer_or_suggester_update_suggestions(handler):
+    """Decorator to check whether user can update the suggestions.
 
     Args:
         handler: function. The function to be decorated.
 
     Returns:
         function. The newly decorated function that has common checks and
-        permissions specified by passed in decorator. Users with rights
-        to accept translations andthe user who created a particular
-        translation are allowed to perform this action.
+        permissions specified by passed in decorator. This function is allowed
+        to be made by the users who have the rights to accept suggestions and
```

Comment:
The wording here seems a bit odd. Perhaps instead: ``is allowed to be called for users who have...`` would be clearer?

Reply:
Done